src/View/Helper/UrlHelper.php

```diff
@@ -280,9 +280,10 @@ public function webroot(string $file): string
         $webPath = $request->getAttribute('webroot') . $asset[0];
         $file = $asset[0];
 
-        if (!empty($this->_View->getTheme())) {
+        $themeName = $this->_View->getTheme();
+        if (!empty($themeName)) {
```

Comment:
Can we use non cloaking, e.g. ! or !== ?

Reply:
Empty string have no value either hence nothing is being cloaked here. The method can only return string or null.

Reply:
Cloaking on variable name. If you misspell $themName etc you get a hidden never true.
I guess PHPStan covers this now, so this is not a big deal, but somewhat still a code smell usually to use cloaking checks (silent isset).

Reply:
Right, use of uninitialized variables is already reported by phpstan and psalm so it's not an issue.